test/test_mps.py

```diff
@@ -9659,6 +9659,12 @@ class TestConsistency(TestCase):
         "true_divide"
     }
 
+    with open("./test/cuda_results.yaml") as f:
```

Comment:
Is this additional file fine? and here it is a relative path from main repo.
@kulinseth @DenisVieriu97